lib/index.js

```diff
@@ -11,6 +11,8 @@ var assertAccessibility = (tagName, props, children, log) => {
 
   var propTests;
   for (var propName in props) {
+    // There's no need to check props that are null or undefined.
```

Comment:
mind killing this comment and then I'll merge? the specs make it pretty clear :)